apps/wpcom-block-editor/src/common/tracking/wpcom-block-picker-search-term-handler.js

```diff
@@ -1,3 +1,5 @@
+/* eslint-disable import/no-extraneous-dependencies */
```

Comment:
Just a quirk of monorepo and `/apps` — as of today needs to be added to every while (you can find other examples).